px-data-grid-date-renderer.html

```diff
@@ -33,7 +55,14 @@
         }
 
         _getFormattedDate(value) {
-          return (new Date(value)).toString();
+          return this.value.length > 0 ? (new Date(value)).toString() : '';
```

Comment:
Can value be undefined?

Reply:
Also new Date(value) will throw exception if not valid? Should this have try catch?

Reply:
Done.